### ISSUE TITLE
[2.8] OptiFine don't launch in 1.13

### ISSUE THREAD

**Issue by jedenastka** [skmedix/SKlauncher#82]
**Describe the bug**
Installed OptiFine 1.13beta4, and it don't launch. Any other OptiFine version work.
Log:
```
[17:18:18] [main/INFO]: Loading tweak class name net.minecraft.launchwrapper.VanillaTweaker
[17:18:18] [main/INFO]: Using primary tweak class name net.minecraft.launchwrapper.VanillaTweaker
[17:18:18] [main/INFO]: Calling tweak class net.minecraft.launchwrapper.VanillaTweaker
Exception in thread "main" java.lang.NoClassDefFoundError: org/objectweb/asm/ClassVisitor
	at java.lang.Class.getDeclaredConstructors0(Native Method)
	at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671)
	at java.lang.Class.getConstructor0(Class.java:3075)
	at java.lang.Class.newInstance(Class.java:412)
	at net.minecraft.launchwrapper.LaunchClassLoader.registerTransformer(LaunchClassLoader.java:88)
	at net.minecraft.launchwrapper.VanillaTweaker.injectIntoClassLoader(VanillaTweaker.java:16)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:115)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
Caused by: java.lang.ClassNotFoundException: org.objectweb.asm.ClassVisitor
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 8 more
```
Launcher log: https://pastebin.com/9AhLRP5W

**To Reproduce**
1.Install OptiFine.
2.Launch "OptiFine" profile
I don't know - maybe it don't work only for me?

**Expected behavior**
Huh, Minecraft 1.13 launching with OptiFine.

**Screenshots**
Logs will be better.

**Environment (please complete the following information):**
 - OS: 
```
grzesiek11@pecet:~$ neofetch
       _,met$$$$$gg.          grzesiek11@pecet 
    ,g$$$$$$$$$$$$$$$P.       ---------------- 
  ,g$$P"     """Y$$.".        OS: Debian GNU/Linux 9.5 (stretch) x86_64 
 ,$$P'              `$$$.     Model: OptiPlex 790 01 
',$$P       ,ggs.     `$$b:   Kernel: 4.9.0-7-amd64 
`d$$'     ,$P"'   .    $$$    Uptime: 7 hours, 31 minutes 
 $$P      d$'     ,    $$P    Packages: 1832 
 $$:      $$.   -    ,d$$'    Shell: bash 4.4.12 
 $$;      Y$b._   _,d$P'      Resolution: 1280x1024 
 Y$$.    `.`"Y$$$$P"'         DE: XFCE 
 `$$b      "-.__              WM: Xfwm4 
  `Y$$                        WM Theme: Arc 
   `Y$$.                      Theme: Arc [GTK2], Adwaita [GTK3] 
     `$$b.                    Icons: Papirus [GTK2], Adwaita [GTK3] 
       `Y$$b.                 Terminal: mate-terminal 
          `"Y$b._             CPU: Intel i5-2400 (4) @ 3.4GHz 
              `"""            GPU: NVIDIA GeForce GTX 660 
                              Memory: 2855MB / 3911MB 
                               
                              ████████████████████████ 
                               

grzesiek11@pecet:~$ 
```
 - Java Version: 
```
grzesiek11@pecet:~$ command java -version
openjdk version "1.8.0_181"
OpenJDK Runtime Environment (build 1.8.0_181-8u181-b13-1~deb9u1-b13)
OpenJDK 64-Bit Server VM (build 25.181-b13, mixed mode)
grzesiek11@pecet:~$ 
```
 - Launcher Version: 2.8.0_41

**Comment by skmedix** [comment]
Try 3.0, it has more fixes on board. But I'll look if some bugfixes are worth to backport

**Comment by jedenastka** [comment]
https://github.com/skmedix/SKlauncher/issues/83
And this is problem with 3.0.

**Comment by skmedix** [comment]
Should be fixed in 2.8.1. (November)

**Comment by Eris7090** [comment]
This is months old but I just had this problem so for anyone else that stumbles upon this thread...

1) Delete the optifine profile from the launcher and the game folder
2) Then reinstall optifine

It's simple and worked for me so I hope it works for anyone else
